index.js

```diff
@@ -1,5 +1,74 @@
 'use strict';
 
+var colorConvert = require('color-convert');
+var supportsColor = require('supports-color');
+
+supportsColor = supportsColor || {};
+supportsColor.level = supportsColor.level || 0;
+
+colorConvert.rgb.rgb = function rgb (rgb) {
+	return rgb;
+};
+
+function getConverter (color) {
```

Comment:
@sunesimonsen We don't do code style linting (yet).

@Qix- It's as easy as `function getConverter (color) {` => `function getConverter(color) {`... We don't do the space between the function's name and the parenthesis.